src/components/PlaylistCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Lesson } from '../store/lessonsSlice';
+import Card from '@material-ui/core/Card';
+
+type PlaylistCardProps = {
+  lesson: Lesson,
+}
+
+const PlaylistCard = ({ lesson }: PlaylistCardProps) => {
```

Comment:
I just realized Kenneth has a component called PlaylistCard too -- might need to rename this